index.bs

```diff
@@ -1223,7 +1219,7 @@ lie outside the valid range of values for its type, as given in
     1.  Let |S| be the sequence of characters matched by the
         <emu-t class="regex"><a href="#prod-float">float</a></emu-t> token.
     1.  Let |result| be the Mathematical Value that would be obtained if
-        |S| were parsed as an ECMAScript [=NumericLiteral=].
+        |S| were parsed as an ECMAScript <emu-nt>[=NumericLiteral=]</emu-nt>.
```

Comment:
Have you checked that this doesn't create issues with the JS script that handles all of the grammar productions dynamically?

Reply:
Yep I've checked. That script doesn't link productions automatically.